src/helpers/helpers.options.js

```diff
@@ -100,9 +99,7 @@ export function _parseFont(options) {
 		family: valueOrDefault(options.fontFamily, defaults.fontFamily),
 		lineHeight: toLineHeight(valueOrDefault(options.lineHeight, defaults.lineHeight), size),
 		size: size,
-		style: valueOrDefault(options.fontStyle, defaults.fontStyle),
-		weight: null,
```

Comment:
So, its not used. Other possibility is to add the option (to docs too)

Reply:
I just wanted to make the minimal changes possible and thought it'd be a bigger change to add a new value and make sure it's added to defaults, properly tested, etc.